packages/react/src/utils/useScrollLock.ts

```diff
@@ -149,11 +149,10 @@ export function useScrollLock(params: {
 }) {
   const { enabled = true, mounted, open, referenceElement = null } = params;
 
-  const isReactAriaHook = React.useMemo(
+  const isBasicLock = React.useMemo(
```

Comment:
A comment with a few words of explanation about basic and standard locks would be nice.